analyzere/extras/visualizations.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+
+from analyzere import LayerView
+from datetime import datetime
+from graphviz import Digraph
+from IPython.display import FileLink
+from sys import float_info
+
+
+class LayerViewVisualizer(object):
+    """Class that provides simple visualization an Analyze Re LayerViews.
+
+    Using the 'graphviz' python package, this class enables users to
+    visualize Analyze Re LayerView objects.
+
+    The class has two main visualization methods:
+
+       view()       open the rendered graph with the default application
+
+       download()   provide a download link to save the rendered graph
+
+
+       Both of these methods provide optional parameters that control the
+       formatting of the graph:
```

Comment:
OK well I'd rather hear another person's opinion and see what they think makes most sense in-case I'm reading it in a weird way. @obaltzer / @sbchisholm ?